src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -1821,4 +1821,7 @@ Please specify one of them using the --source option to proceed.</value>
     <value>Reboot required to fully enable the Windows Feature(s); proceeding due to --force</value>
     <comment>"Windows Feature(s)" is the name of the options Windows features setting.</comment>
   </data>
+  <data name="InstallWaitingOnAnother" xml:space="preserve">
+    <value>Waiting on another install or uninstall to complete...</value>

```

Comment:
Nit: Shouldn't it be "Waiting for"?

Reply:
Turns out that my southern US origins may be the source of this based on a brief look.  They both are "correct" but various sources give differing opinions, but that one vs the other is colloquial.  I have no problem changing it though.